ceph.spec.in

```diff
@@ -2128,6 +2128,8 @@ fi
 %if %{with lttng}
 %{_libdir}/librbd_tp.so.*
 %endif
+%dir %{_libdir}/ceph/librbd
+%{_libdir}/ceph/librbd/libceph_*.so*
```

Comment:
Do we really need `ceph/` subdir in the path? E.g. the package is called "librbd", not "ceph-librbd".

Reply:
The `common/PluginRegistry` requires all plugins to start with `libceph_` in a subdirectory of the `lib[64]/ceph` directory. I added an extra `librbd_` in the middle (i.e. `/usr/lib64/ceph/libceph/librbd/libceph_librbd_parent_cache.so`) just to prevent any possible collision during the build stage. The crypto plugins are similar w/ `/usr/lib64/ceph/crypto/libceph_crypto_XYZ.so`).

I did think about using `rbd` instead of `librbd`, but thought it would be better to match the actual library and package name.

Reply:
ack